data/panel.css

```diff
@@ -229,7 +230,6 @@ input[type=range]::-moz-focus-outer {
 
 .video-progress {
     background: #ffffffaa;
-    border-radius: 20px;
```

Comment:
Here's what it looks like with padding-left. A little better, but still not quite right.
<img width="73" alt="screen shot 2016-08-29 at 4 47 28 pm" src="https://cloud.githubusercontent.com/assets/9204835/18100855/1f78f56e-6eb2-11e6-9646-50ee14e1a243.png">